### PR TITLE
improvement: identify primary keys as such

### PR DESCRIPTION
**Pull request by ghusse**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [x] Create automatic tests
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

**Review by arnaudbesnier — DISMISSED** [review]
You have a conflict, you need to rebase.
But looks good to me!

File: test/adapters/sequelize.test.js [hunk 1]
```diff
@@ -75,6 +75,7 @@ function getField(schema, name) {
             field: 'username',
             type: 'String',
             columnName: 'username',
+            isPrimaryKey: false,
```
Comment (arnaudbesnier) [inline test/adapters/sequelize.test.js:78]: As you can see here the `username` field does not contain all the properties ; only `field`/`type`/`isPrimary` and not `isRequired`/`isFilterable`/`isVirtual`/...
It is not the concern of the adapter to aggregate the "default" values of all properties, I think it is currently done in `forest-express` dependency when the `.forestadmin-schema.json` content is serialized.

File: src/services/apimap-field-builder.js [hunk 2]
```diff
@@ -143,9 +143,8 @@ function ApimapFieldBuilder(model, column, options) {
...
     };
 
-    if (column.primaryKey === true) {
-      schema.primaryKey = true;
-    }
+    schema.isPrimaryKey = Boolean(column.primaryKey);
+
     if (schema.type === 'Enum') {
       schema.enums = column.values;
     }
```
Comment (arnaudbesnier) [inline src/services/apimap-field-builder.js:150]: `enums` / `isRequired` properties are not always set but we can retrieve them in each field definition in the `.forestadmin-schema.json`.

**Review by arnaudbesnier — COMMENTED** [review]
See my 2 related comments.

File: test/adapters/sequelize.test.js (hunk 1 above)
Comment (ghusse) [reply test/adapters/sequelize.test.js]: Ok, I'll make sure that the property is always set by `forest-express`.

**Review by arnaudbesnier — APPROVED** [review]
🚀

**Pull request merged**